src/streaming/SourceBufferSink.js

```diff
@@ -345,6 +345,18 @@ function SourceBufferSink(mediaSource, mediaInfo, onAppendedCallback, useAppendW
         }
     }
 
+    function pollHandler(handler, interval) {
```

Comment:
Can we move pollHandler and stopPollHandler to the Utils class? Otherwise we will have to copy the same two functions to each class which wants to use setInterval.